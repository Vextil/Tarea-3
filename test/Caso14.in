# stress ColaBinario
# Prueba de orden de tiempo
TiempoColaBinario
# Prueba de uso de memoria
MemoriaColaBinario
Fin
