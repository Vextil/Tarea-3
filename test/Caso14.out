1># stress ColaBinario
2># Prueba de orden de tiempo
3>Fin Encolar. 
4># Prueba de uso de memoria
5>Fin Destruir ColaBinario. Fin DesencolarBinario. 
6>Fin
